### DISCUSSION TITLE
Multiple Dropdown toggles but open only one

### DISCUSSION THREAD

**Discussion by MrVibe** [saadeghi/daisyui#3079] [Q&A]
I am using the react-version and I want to use multiple dropdown toggles but activate only 1 at a time. The toggle should also close by clicking outside. 

The behaviour imitates a radio button feature, I already tried below but it did not work as expected.

````
 <Dropdown.Details end>
                        <Dropdown.Details.Toggle color="ghost">
                          <label for={page.id.toString()}><EllipsisVerticalIcon className="h-6 w-6" /></label>
                          <input type="radio" id={page.id.toString()} name="pages" />
                        </Dropdown.Details.Toggle>
                        <Dropdown.Menu className="w-52 z-50">
````
anyone has better ideas on how to achieve this.

**Comment by saadeghi** [comment]
This repo is for daisyUI.

Ask questions about `react-daisyui` on its own repo please:  
https://github.com/daisyui/react-daisyui
